src/app/producer-shelf/components/tv2-action-card/tv2-action-card.component.scss

```diff
@@ -52,4 +57,8 @@
   &:active {
     opacity: 0.7;
   }
+
+  .titleContainer {
```

Comment:
We use kebab-case for CSS classes: `title-container`.

Reply:
Good catch.
This has been changed to just `title` after implementing your other comment.